server/cli/launch.py

```diff
@@ -26,14 +26,14 @@
 
 def annotation_args(func):
     @click.option(
-        "--experimental-annotations",
+        "--disable-annotations",
```

Comment:
I can't find a test case to test this feature (required for non-experimental features).  Please add at least a simple test that the system works correctly with annotations disabled.

Reply:
Will do, thanks for the catch